bin/pa11y-ci.js

```diff
@@ -199,6 +199,11 @@ function defaultConfig(config) {
 	return config;
 }
 
+// Generate a file name string from url
+function getFileNameFromUrl(url) {
+	return url.trim('/').split('/').join('-').replace('.', '').replace(':', '');
```

Comment:
`.replace(':', '')` is un-necessary. `:` are fine in a filename

Reply:
In fact, use https://github.com/sindresorhus/filenamify to catch all the edge cases of a URL not beingn a safe filename

Reply:
Hi @joeyciechanowicz,

Many thanks for your comments. I have used filenamify as you said and replace the line.